python/py-ipympl/Portfile

```diff
@@ -21,16 +23,44 @@ long_description    Leveraging the Jupyter interactive widgets framework, \
 
 homepage            https://github.com/matplotlib/jupyter-matplotlib
 
-checksums           rmd160  edabf6803acf97df26b7fe0b24483da64bee70c5 \
-                    sha256  f0f1f356d8cb9d4fb51bb86dbbf837c190145316cb72f66081872ebc4d6db0a1 \
-                    size    56729
+checksums           rmd160  7524cb752caa0423a00cc15b6ec6881a0961c774 \
+                    sha256  49bab75c05673a6881d1aaec5d8ac81d4624f73d292d154c5fb7096f10236a2b \
+                    size    267567
 
 if {${name} ne ${subport}} {
     depends_build-append \
                         port:py${python.version}-setuptools \
+                        port:py${python.version}-wheel \
                         port:py${python.version}-jupyter_packaging
 
-    depends_lib-append  port:py${python.version}-ipykernel \
+    depends_lib-append  \
+                        port:py${python.version}-jupyterlab \
+                        port:py${python.version}-ipython \
+                        port:py${python.version}-numpy \
+                        port:py${python.version}-ipython_genutils \
+                        port:py${python.version}-pillow \
+                        port:py${python.version}-traitlets \
                         port:py${python.version}-ipywidgets \
                         port:py${python.version}-matplotlib
```

Comment:
where do all the additional dependencies come from? The only ones listed in `setup.py` are:
```
    install_requires=[
        'ipython<9',
        'numpy',
        'ipython_genutils',
        'pillow',
        'traitlets<6',
        'ipywidgets>=7.6.0,<9',
        'matplotlib>=3.4.0,<4',
    ],
```

Reply:
also the correct name is `py-Pillow` with the capital "P" - that should be used to avoid issues on case-sensitive file systems.

Reply:
The only "extra" one is jupyterlab. That comes from pyproject.toml. I'll have a look at this again in a few days. If I remember correctly, then py37 had problems with the upgrade.